src/devices/Display/samples/Program.Matrix16x8.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
maybe update the main README.md to add some of those elements?

Reply:
Good idea. Will do that after the Bicolor PR merges.